tests/hypothesis_strategies/kernels.py

```diff
@@ -21,8 +33,8 @@
 
 
 @st.composite
-def kernels(draw: st.DrawFn):
-    """Generate :class:`baybe.kernels.basic.Kernel`."""
+def single_kernels(draw: st.DrawFn):
```

Comment:
The test should also be called `simple_kernels` such that it aligns with the docstring.

Reply:
Thanks, I instead adjusted the docstring 👍🏼